File: src/AspNetCoreAngular2/wwwroot/app/models/RDSignature.ts
import { RD } from '../models/RD';

export class RDSignature extends RD {

	public Signature: string;
	public UniqueStamp: string;

	constructor(rdUserName: string, signalRClientId: string, staticClientGuid: any, signature: string, uniqueStamp: string) {
		super(rdUserName, signalRClientId, staticClientGuid);

		this.Signature = signature;
		this.UniqueStamp = uniqueStamp;
	}

}

File: src/AspNetCoreAngular2/Models/RDSignature.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ASPNETCoreAngular2Demo.Models
{
    public class RDSignature : RD
    {
		public string Signature { get; set; }

		/// <summary>
		/// This must match the signaturescreendetail value.
		/// </summary>
		public string UniqueStamp { get; set; }
	}
}
